Makefile

```diff
@@ -417,13 +417,13 @@ KUBE_CHART_VERSION          ?= ">=1.25.0-0"
 codegen-api-register: ## Generate API types registrations
 codegen-api-register: $(REGISTER_GEN)
 	@echo Generate registration... >&2
-	@$(REGISTER_GEN) --go-header-file=./scripts/boilerplate.go.txt --output-file zz_generated.register.go ./api/...
+	@$(REGISTER_GEN) --go-header-file=./scripts/boilerplate.go.txt --output-file-base zz_generated.register ./api/...
```

Comment:
--output-file-base doesn't exist in newer version of the tooling.

run `make clean-tools` and `make codegen-all` it should fix your issue